app/models/auction_query.rb

```diff
@@ -98,7 +98,7 @@ def public_find(id)
       .find(id)
   end
 
-  def my_bids(user_id)
+  def user_has_bid(user_id)
```

Comment:
maybe `user_bids` would be a more apt name for this given that it returns a collection of bids for a user?